package.json

```diff
@@ -19,17 +19,22 @@
     "url": "git://github.com/jameslnewell/xhr-mock.git"
   },
   "dependencies": {
+    "browserify": "^11.0.1",
     "global": "^4.3.0",
+    "lie": "^3.1.1",
+    "uglify-js": "^2.7.5",
     "url-parse": "^1.1.7"
   },
   "devDependencies": {
-    "browserify": "^14.0.0",
     "mochify": "^2.13.0",
     "superagent": "^3.4.1"
   },
   "scripts": {
     "test": "mochify",
-    "example.build": "mkdir -p build && browserify -r superagent -r ./index.js:xhr-mock -o build/build.js"
+    "build": "mkdir -p dist && browserify -r lie -r ./index.js:xhr-mock -o dist/xhr-mock.js",
+    "build:example": "browserify -r lie -r superagent -r ./index.js:xhr-mock -o example/xhr-mock.js",
+    "minify": "uglifyjs dist/xhr-mock.js --output dist/xhr-mock.min.js",
```

Comment:
I use selenium, so I inject script into page. I would like to minify it, so it will speedup. 

I think superagent required only for examples, not for everyone.